### PR TITLE
Remove connection-ui Package

### PR DESCRIPTION
**Pull request by oskosk**
Removes unused package that used to serve as an example consumer of the connection package and the js-package/connection module.

Not needed anymore since we built My Jetpack and serves a similar purpose

#### Changes proposed in this Pull Request:

* Removes the directory packages/connection-ui
* Updates composer and lock files for Jetpack, Backup and the Backup packge
* Updates pnpm lock files as the JS from connection-ui is not consumed anywhere
* Removes mentions to ConnectionUI in some initialization code comments.
* Removes the `connection-ui-activator.php` mu-plugin from the dev env.

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion

p1663941833670189-slack-C02NENKA4GN

p1643118545097400/1643118147.096500-slack-C02TQF5VAJD

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
Confirm all builds in this PR succeed

### CODE REVIEW

File: projects/js-packages/connection/README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ const statusCallback = useCallback(
 	apiRoot="https://example.org/wp-json/" 
 	apiNonce="12345"
 	registrationNonce="54321"
-	from="connection-ui"
+	from="my-jetpack"
```
Comment (sergeymitr) [inline projects/js-packages/connection/README.md:59]: We should probably replace it with something generic so we wouldn't mess up our stats when somebody copies the code without replacing the `from` keys.
What do you think?

**Review by sergeymitr — DISMISSED** [review]
Thanks for taking care of this!

I also noticed some calls to the `ConnectionUI` class are still there. If you need help cleaning those up, please let me know.

File: projects/js-packages/connection/README.md (hunk 1 above)
Comment (oskosk) [reply projects/js-packages/connection/README.md:59]: I don't think so. This is real usage and people can grep the code for and see the real-world implementation. Coming up with an abstract name won't add much for a person's purpose of just making it work

**Comment by oskosk** [comment]
> I also noticed some calls to the ConnectionUI class are still there. If you need help cleaning those up, please let me know.

@sergeymitr I think I addressed this in 63118fcdde now

**Comment by oskosk** [comment]
I'm' getting this from the tests. Not sure how to proceed

image

**Review by sergeymitr — APPROVED** [review]
Thanks for taking care of this 👍

**Comment by oskosk** [comment]
Thanks Sergey

**Comment by oskosk** [comment]
r253142-wpcom

**Pull request merged**